packages/@aws-cdk/aws-ec2/lib/vpc-flow-logs.ts

```diff
@@ -175,6 +176,13 @@ export interface FlowLogDestinationConfig {
    * @default - undefined
    */
   readonly s3Bucket?: s3.IBucket;
+
+  /**
+   * S3 bucket subfolder to publish the flow logs to
+   *
+   * @default - undefined
+   */
+  readonly subfolder?: string;
```

Comment:
Lets keep the AWS S3 convention and use `Prefix`

Reply:
Updated to `keyPrefix`